myjsonrpc.pm

```diff
@@ -14,6 +14,10 @@ use bmwqemu ();
 use constant DEBUG_JSON => $ENV{PERL_MYJSONRPC_DEBUG} || 0;
 use constant READ_BUFFER => $ENV{PERL_MYJSONRPC_BYTES} || 8_000_000;
 
+sub get_written_bytes($to_fd, $json) {
+    return syswrite($to_fd, "$json");
+}
```

Comment:
```suggestion
sub _syswrite($to_fd, $json) { syswrite($to_fd, "$json") }
```

Reply:
And also avoid the string interpolation:

```suggestion
sub _syswrite($to_fd, $json) { syswrite($to_fd, $json) }
```

We could actually skip the argument processing here to make it even simpler and more lightweight:

```suggestion
sub _syswrite(@_) { syswrite(@_) }
```